content/pt/docs/2.concepts/5.nuxt-lifecycle.md

```diff
@@ -1,108 +1,108 @@
+- O processo de geração inicia com comando (`nuxt generate`)
 
-- Nuxt hooks
+- Gatilhos do Nuxt
 - serverMiddleware
-- Server-side Nuxt plugins
-  - in order as defined in nuxt.config.js
+- extensões da Nuxt do lado do servidor
+  - conforme foi definido no `nuxt.config.js`
 - nuxtServerInit
-  - Vuex action that is called only on server-side to pre-populate the store
-  - First argument is the **Vuex context**, second argument is the **Nuxt context**
-    - Dispatch other actions from here → only "entry point" for subsequent store actions on server-side
-  - can only be defined in `store/index.js`
-- Middleware
-  - Global middleware
-  - Layout middleware
-  - Route middleware
+  - A ação da Vuex que é chamada apenas no lado do servidor para pré-popular a memória
+  - O primeiro argumento é o **contexto da Vuex**, o segundo argumento é o **contexto da Nuxt**
+    - Despacha outras ações a partir de cá → apenas no "ponto de entrada" para ações da memória subsequente no lado do servidor
+  - apenas pode ser definida no `store/index.js`
+- Intermediário (Middleware)
+  - Intermediário Global
+  - Intermediário de Esquema
+  - Intermediário de Rota
 - asyncData
-- beforeCreate (Vue lifecycle method)
-- created (Vue lifecycle method)
-- The new fetch (top to bottom, siblings = parallel)
-- Serialization of state (`render:routeContext` Nuxt hook)
-
-- the HTML rendering happens (`render:route` Nuxt hook)
-
-- `render:routeDone` hook when HTML has been sent to the browser
-
-- `generate:before` Nuxt hook
-- HTML files are generated
-  - **Full static generation**
-    - e.g. static payloads are extracted
-  - `generate:page` (HTML editable)
-  - `generate:routeCreated` (Route generated)
-- `generate:done` when all HTML files have been generated
-
-### Client
-
-This part of the lifecycle is fully executed in the browser, no matter which Nuxt mode you've chosen.
-
-- Receives the HTML
-- Loading assets (e.g. JavaScript)
-- client-side Nuxt plugin
-  - in order as defined in nuxt.config.js
-- Vue Hydration
-- Middleware
-  - Global middleware
-  - Layout middleware
-  - Route middleware
-- asyncData (blocking)
-- beforeCreate (Vue lifecycle method)
-- created (Vue lifecycle method)
-- The new fetch (top to bottom, siblings = parallel) (non-blocking)
-- beforeMount (Vue lifecycle method)
-- mounted (Vue lifecycle method)
-
-### Navigate using the NuxtLink component
-
-Same as for the _client_ part, everything is happening in the browser but only when navigating via `<NuxtLink>`. Furthermore, no page content is displayed until all _blocking_ tasks are fulfilled.
+- beforeCreate (método do ciclo de vida da Vue)
+- created (método do ciclo de vida da Vue)
+- O novo `fetch` (de cima para baixo, irmãos = paralelo)
+- Adaptação de estado (gatilho `render:routeContext` da Nuxt)
+
+- A interpretação do HTML acontece (gatilho `render:route` da Nuxt)
+
+- `render:routeDone` aciona quando o HTML é enviado para o navegador
+
+- Gatilho `generate:before` do Nuxt
```

Comment:
I've you use `o Nuxt` e `a Nuxt`, please stick to the non gender form

Reply:
@ChristopheCVB it is impossible, there are no ways to write
about some tool, person, animal, or thing in a more objective
way in Portuguese without using any definite article. :sweat_smile: 

I am using `a` because I am presenting the Nuxt as "a tool",
translated as "uma ferramenta" and as "a framework",
translated as "uma abstração".

Portuguese is not English, and all these chances are to put the
things to sound a little more Portuguese. :point_up:

Reply:
If you think not using a gender is not correct, then fallback to the masculine.
Here is a nice post about genders in portuguese https://rockcontent.com/br/talent-blog/linguagem-neutra-de-genero/